src/main/java/com/shippingcostestimator/enterprise/dto/PackageInfo.java

```diff
@@ -0,0 +1,14 @@
+package com.shippingcostestimator.enterprise.dto;
+
+import lombok.Data;
+
+public @Data
+class PackageInfo {
+    //PARCEL
+    private double length;
+    private double width;
+    private double height;
+    private String predefinedPackage;
+    private double weight;
+}
+
```

Comment:
I took this and made its own class.  https://metova.com/how-to-evaluate-manage-and-avoid-technical-debt/
“God objects” is a term used for technical debt. Where an object seems to take on the responsibility of far too many things. This should help eliminate that. Also, if you end up wanting to add the ability to have more than one package calculated. This would help.